.gitignore

```diff
@@ -109,6 +109,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+.env
```

Comment:
I had to revert your PR in https://github.com/openfoodfacts/open-prices/pull/787

Reply:
the file is used in the Makefile

Reply:
the CI was raising errors...
```
api Pulling 
api Error manifest unknown
Error response from daemon: manifest unknown
```

Reply:
not sure how to fix this in the long run though..

Reply:
fyi in my local environment, I run the app without docker, passing in the env variables as prefixes in the terminal
```
POSTGRES_DB=open_prices POSTGRES_USER=... POSTGRES_PASSWORD=... POSTGRES_HOST=... ALLOWED_HOSTS=localhost,127.0.0.1 CSRF_TRUSTED_ORIGINS=http://127.0.0.1:8000 SECRET_KEY=... OAUTH2_SERVER_URL=https://world.openfoodfacts.org/cgi/auth.pl DEBUG=True poetry run python manage.py runserver
```